microceph/common/fileUtils.go

```diff
@@ -0,0 +1,43 @@
+package common
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	"github.com/canonical/lxd/shared/logger"
+	"github.com/djherbis/times"
+)
+
+// FilterFilesInDir filters filenames which matches the substring in path.
+func FilterFilesInDir(subString string, path string) []string {
```

Comment:
Possibly could use the stdlib function `filepath.Glob()` to do this
https://pkg.go.dev/path/filepath#Glob